ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -1303,6 +1303,27 @@ tests:
     - chain: ipi-install
     - ref: ipi-azure-rbac
   timeout: 6h0m0s
+- as: gcp-private
+  optional: true
+  run_if_changed: (gcp|google)
+  steps:
+    cluster_profile: gcp
+    env:
+      PUBLISH: Internal
+    post:
+    - ref: gather-installer-boostrap-logs-in-bastion
+    - chain: ipi-gcp-post
```

Comment:
Missing `gcp-deprovision-bastionhost` and `gcp-deprovision-vpc`

Reply:
Got it. 

Sorry, I forgot why we are not using the existing chains `cucushift-installer-rehearse-gcp-ipi-private-provision` & `cucushift-installer-rehearse-gcp-ipi-private-deprovision` instead, do you happen to have any idea?

Reply:
For me, either is okay for me.

Reply:
Ok, let me try, thanks!